OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -142,3 +147,33 @@ void testVisModel(string fileName)
     delete model;
     ASSERT(same == 0, __FILE__, __LINE__, "Files do not match.");
 }
+
+void createModel4AppearanceTest(const std::string& filename)
+{
```

Comment:
This model creation needs to be better documented so that there is unambiguous correspondence between the visualization of Geometry (explicit and implied) and the vis file standard below.  Describe every piece of geometry and how it should appear (what transform, rep, opacity, scale) it should have. It should not be up to the reviewer and future tester to deduce that correspondence.